pydis_core/utils/error_handling/commands/manager.py

```diff
@@ -0,0 +1,61 @@
+from collections.abc import Callable
+from typing import NoReturn
+
+from discord import Interaction
+from discord.ext.commands import Context
+
+from pydis_core.utils.error_handling.commands import AbstractCommandErrorHandler
+from pydis_core.utils.logging import get_logger
+
+log = get_logger(__name__)
+
+
+class CommandErrorManager:
+    """A class that registers error handlers and handles all command related errors."""
+
+    def __init__(self, default: AbstractCommandErrorHandler, handlers: list[AbstractCommandErrorHandler] | None = None):
+        self.handlers = handlers or []
+        self.registered_handlers = set(type(handler).__name__ for handler in self.handlers)
+        self.default = default
```

Comment:
Should these be "private" with `_` prefix?

Reply:
Definitely, I always forget to do that since there are no access modifiers.